pt/git.md

```diff
@@ -37,9 +37,12 @@ Maus exemplos seriam: `maria_22_de_abril`, `regras`.
 
 ## [Mensagens de *commit*](https://xkcd.com/1296/)
 
+Use [estas diretrizes](https://cbea.ms/git-commit/) (em inglês).
+
 Dentro do possível, mantenha seus *commits* bem descritos:
 - comece todos as suas mensagens de *commit* com a tag `[pt]`;
-- prefira escrevê-las em inglês;
-- a primeira linha da sua mensagem não deve ultrapassar 50 caracteres.
+- escreva-as em inglês;
+- a primeira linha da sua mensagem não deve ultrapassar 50 caracteres;
+- a não ser que suas mudanças sejam gerais (ou seja, se apliquem a várias regras), busque pôr o nome da regra na primeira linha da mensagem.
 
-O adágio "*commit early, commit often*" só se aplica ao seu trabalho **local**.
+O adágio "*commit early, commit often*" só se aplica ao seu trabalho **local**. Ao criar um *pull request*, o ideal é que ele seja composto de poucos *commits* razoavelmente completos. Prefira fazer *squash* se você tiver muitos *commits* com mensagens do tipo `"[pt] Fix error"`, `"[pt] Add antipattern"`.
```

Comment:
Alternatively, with precise instructions: 

> O adágio "*commit early, commit often*" só se aplica ao seu trabalho **local**. Ao criar um *pull request*, o ideal é que ele seja composto de poucos *commits* razoavelmente completos. Prefira fazer *squash* se você tiver muitos *commits* com mensagens do tipo `"[pt] Fix error"`, `"[pt] Add antipattern"` **(isto é, no botão "Merge pull request", dê preferência à opção "Squash and merge").**

Reply:
Well, that's not the only way to squash. We can (as I frequently do) squash locally, earlier than at the PR stage.

Reply:
Another reason why perhaps we should make the process more explicit/descriptive? So that less knowledgeable people know their options/how to do it.

Reply:
I mean, sure, but explaining Git procedures isn't really within the scope of this documentation. What we *could* do is add a link to some other page explaining the concept of 'squashing'?